gradle-nonnull-plugin/plugin/src/test/groovy/com/novoda/gradle/nonnull/AndroidProjectIntegrationTest.groovy

```diff
@@ -0,0 +1,62 @@
+package com.novoda.gradle.nonnull
+
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.internal.DefaultGradleRunner
+import org.junit.ClassRule
+import org.junit.Test
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+import static com.google.common.truth.Truth.assertThat
+
+public class AndroidProjectIntegrationTest {
+
+    @ClassRule
+    public static final ProjectRule PROJECT = new ProjectRule()
+
+    @Test
+    public void shouldGeneratePackageAnnotation() {
+        PROJECT.generatedSrcDirs.each {
+            def file = new File(it, 'package-info.java')
```

Comment:
When the variable is used locally, the IDE autocompletion works because it infers the type, no? Especially in the tests, I thought this is more groovy because writing `File` twice is redundant.